Packs/IllusiveNetworks/Integrations/IllusiveNetworks/IllusiveNetworks.py

```diff
@@ -720,6 +882,22 @@ def main():
         elif demisto.command() == 'illusive-run-forensics-on-demand':
             return_outputs(*run_forensics_on_demand_command(client, demisto.args()))
 
+        elif demisto.command() == 'illusive-get-forensics-artifacts':
+            try:
+                return_results(get_forensics_artifacts_command(client, demisto.args()))
+                return_results(link_forensics_artifacts_name_command(True, client, demisto.args()))
+            except ValueError:
```

Comment:
what value error are you expecting here?

Reply:
there is only one value error that can be thrown raise ValueError("file is empty") - line 759